include/RAJA/policy/desul/atomic_desul_openmp.hpp

```diff
@@ -0,0 +1,184 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef RAJA_policy_atomic_desul_openmp_HPP
+#define RAJA_policy_atomic_desul_openmp_HPP
+
+#include "RAJA/config.hpp"
+
+#if defined(RAJA_ENABLE_OPENMP)
+
+#include "RAJA/util/macros.hpp"
+
+#include "RAJA/policy/atomic_builtin.hpp"
+
+#include "desul/atomics.hpp"
+
+// Default desul options for RAJA
+using raja_default_desul_order = desul::MemoryOrderRelaxed;
+using raja_default_desul_scope = desul::MemoryScopeDevice;
+
+namespace RAJA
+{
+
+#if defined(RAJA_COMPILER_MSVC)
+
+// For MS Visual C, just default to builtin_atomic for everything
+using omp_atomic = builtin_atomic;
```

Comment:
This line here seems to cause errors as far as I can tell, seems to result in `builtin_atomic` being undefined.